javaschool-backend/src/main/java/com/nearsoft/javaschoolbackend/model/request/TypeRequest.java

```diff
@@ -0,0 +1,26 @@
+package com.nearsoft.javaschoolbackend.model.request;
+
```

Comment:
tip: Take a look to lombok library to avoid boilerplate code.
[https://projectlombok.org/](url)